<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crudCulturasDialog</class>
 <widget class="QDialog" name="crudCulturasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Culturas</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>22</y>
     <width>583</width>
     <height>266</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>0</height>
         </size>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Info</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="novaCulturaButton">
          <property name="text">
           <string>Nova Cultura</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="novaConfButton">
          <property name="text">
           <string>Nova Configuração Regional</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="novoGrupoButton">
          <property name="text">
           <string>Novo Grupo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editarItemButton">
          <property name="text">
           <string>Editar Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removerItemButton">
          <property name="text">
           <string>Remover Item</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>crudCulturasDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>crudCulturasDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
